### PR TITLE
Update strings.xml Hebrew language

### PR DESCRIPTION
**Pull request by htht2001**
Translation update to the Hebrew language

### CODE REVIEW

File: app/src/main/res/values-iw/strings.xml [hunk 1]
```diff
@@ -18,47 +21,102 @@
...
-        <item quantity="one">שמור %d יום</item>
-        <item quantity="other">שמור %d ימים</item>
+        <item quantity="one">לשמור %d יום</item>
+        <item quantity="other">לשמור %d ימים</item>
     </plurals>
-    -->
+    <string name="retention_unusable">שמירת קבצים אינה זמינה מכיוון שתבנית שם הקובץ הנוכחית אינה תואמת לתכונה.</string>
 
-    <!-- Output format bottom sheet -->
+        Output format bottom sheet -->
```
Comment (htht2001) [inline app/src/main/res/values-iw/strings.xml:59]: `<!-- Output format bottom sheet -->`

**Comment by htht2001** [comment]
I basically fixed the problem
Can you verify that?
Thank you

**Review by chenxiaolong — APPROVED** [review]
Thanks!

**Pull request merged**